pages/common/exiftool.md

```diff
@@ -5,7 +5,7 @@
 
 - Print the EXIF metadata for a given file:
 
-`exiftool {{file}}`
+`exiftool {{path/to/file}}`
 
 - Remove all EXIF metadata from the given files:
 
```

Comment:
Line 12 must be `exiftool -All= {{path/to/file1 path/to/file2 ...}}`

Reply:
Fixed.